.github/workflows/helm-chart-lint.yml

```diff
@@ -1,18 +1,43 @@
 name: Lint Helm Charts
 
-on: pull_request
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "helm/provisioner/**"
 
 jobs:
   lint-test:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
```

Comment:
were there issues pinning a tagged release?

Reply:
Tags can be moved, so shouldn't be used. Using commits directly in combination with Dependabot to generate the PRs (these show the versions & change log) is the recommended way to work with GH Actions for a secure supply chain.